package.json

```diff
@@ -11,12 +11,14 @@
   },
   "homepage": "https://github.com/Universal-Connect-Project/ucp-hosted-app#readme",
   "scripts": {
-    "build": "if [ -z \"$UCP_UI_REVIEW_APP\" ]; then npm run build:main; else npm run build:review; fi",
+    "build": "./build.sh",
     "build:review": "npm run heroku:review:setupEnv && turbo build",
-    "build:main": "turbo build",
-    "build:prod": "turbo build:prod",
+    "build:staging": "npm run heroku:staging:setupEnv && turbo build",
```

Comment:
I didn't add this to turbo because it does what I need it to do with `turbo build`. The thing that's different is the environment variables set the institution and authentication service URLS to point to our heroku deployed services.